src/tools/Reporting/Reporting/Counter.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Reporting
+{
+    public class Counter
+    {
+        public string Name { get; set; }
```

Comment:
nit: if `Build` is sealed, should this type be too?

Reply:
Heh, I didn't actually intend to seal build. I'll make a note to clean that up at some point.